tsl/test/src/test_compression.c

```diff
@@ -281,15 +280,14 @@ test_gorilla_float()
 	GorillaCompressor *compressor = gorilla_compressor_alloc();
 	GorillaCompressed *compressed;
 	DecompressionIterator *iter;
-	float i;
-	for (i = 0.0; i < TEST_ELEMENTS; i++)
-		gorilla_compressor_append_value(compressor, float_get_bits(i));
+	for (int i = 0.0; i < TEST_ELEMENTS; i++)
+		gorilla_compressor_append_value(compressor, float_get_bits((float) i));
 
 	compressed = gorilla_compressor_finish(compressor);
 	TestAssertTrue(compressed != NULL);
 	TestAssertInt64Eq(VARSIZE(compressed), 1200);
 
-	i = 0;
+	float i = 0;
```

Comment:
Suggest to use a different variable name to avoid confusion when reading the code.

Reply:
Sorry, I had auto-merge enabled, I'll add it into the text bulk decompression PR.